commcare_connect/opportunity/tests/test_api_views.py

```diff
@@ -235,7 +236,7 @@ def test_delivery_progress_endpoint(
     assert response.data.keys() == DeliveryProgressSerializer().get_fields().keys()
     assert len(response.data["deliveries"]) == 1
     assert len(response.data["payments"]) == 0
-    assert response.data["deliveries"][0].keys() == UserVisitSerializer().get_fields().keys()
+    assert response.data["deliveries"][0].keys() == CompletedWorkSerializer().get_fields().keys()
```

Comment:
This is a good catch. It was orginally like that because we switched serializers and wanted to ensure compatibility, but I agree that now that we are making additions, we should use the correct one. That said, I am not sure this assertion is buying us much, and would prefer the test to check expected values of keys we care about in the fields rather than just the presence of the keys.

Reply:
True true. I'll update!